src/main/java/seedu/address/logic/commands/MeetingsCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.SameMeetingDayPredicate;
+
+//@@author AyushChatto
+/**
+ * Returns details of meetings scheduled at a particular time.
+ */
+public class MeetingsCommand extends Command {
+    public static final String COMMAND_WORD = "meetings";
+    public static final String COMMAND_ALIAS = "mtngs";
```

Comment:
Why not use the alias 'm' instead? It might be hard for the user to remember 'mtngs' and they might type it wrongly.

Reply:
Hey Denzel, I wanted to avoid confusion between the prefix for meetings 'm/' and the alias, but since it shouldn't be a problem, I've changed it. Cheers!